src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -1155,7 +1155,7 @@ fatal e = do
     rk <- view txRequestKey
 
     logError_ l
-      $! "critical transaction failure: "
+      $ "critical transaction failure: "
```

Comment:
What's the reason for undoing the bangs

Reply:
Loggers are always strict, so the bangs aren't necessary... except that they're lazy when they're filtered such that the log message never makes it to the queue. I've seen some pretty big log messages here, so I'd like those filtered out in case that can raise throughput.

Reply:
yeah, log messages are deep forced before added to the queue.

Avoiding construction when their log level is disabled is a good point.